dev-docs/how-prebid-on-amp-works.md

```diff
@@ -13,7 +13,7 @@ nav_section: prebid-amp
 # How Prebid on AMP Works
 {: .no_toc}
 
-{: .pb-img.pb-md-img :}
+{: .pb-img.pb-xl-img :}
```

Comment:
These no such class `pb-xl-img` in our styles.css. You can remove this class. It aligns properly without it.